### ISSUE TITLE
Compute: No MLCTrainingGraph has been found.

### ISSUE THREAD

**Issue by vicky8989** [bubbliiiing/yolov4-tiny-tf2#14]
enviroment：macos tensorflow 2.4.0-rc0 ，python3.8
problem: get error, Compute: No MLCTrainingGraph has been found.

is my tensorflow version not match? otherwise gpu mode is failed? do anyone meet the same problem?

i can run some test code,like bellow:

import tensorflow as tf
import time

 # Import mlcompute module to use the optional set_mlc_device API for device selection with ML Compute.
from tensorflow.python.compiler.mlcompute import mlcompute
mlcompute.set_mlc_device(device_name = 'gpu')

mnist = tf.keras.datasets.mnist

(x_train, y_train),(x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='relu'),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

start = time.time()

model.fit(x_train, y_train, epochs=10)

end = time.time()

model.evaluate(x_test, y_test)
print("time consumed: ", end - start)
